libs/editor/libs/editor-common/assets/ZSSRichTextEditor.js

```diff
@@ -1138,45 +1144,76 @@ ZSSEditor.removeImage = function(imageNodeIdentifier) {
  *  @brief Inserts a video tag using the videoURL as source and posterURL as the
  *  image to show while video is loading.
  *
- *  @param videoURL  the url of the video to present
- *  @param posterURL the url of an image to show while the video is loading
- *  @param alt       the alt description when the video is not supported.
+ *  @param videoURL     the url of the video
+ *  @param posterURL    the url of an image to show while the video is loading
+ *  @param videoPressID the VideoPress ID of the video, when applicable
  *
  */
-ZSSEditor.insertVideo = function(videoURL, posterURL, alt) {
-    var html = '<video controls webkit-playsinline poster="' + posterURL+ '"><source src="'
-    + videoURL + '" type="video/mp4"/>' + alt + '</video>';
+ZSSEditor.insertVideo = function(videoURL, posterURL, videopressID) {
+    var html = '<video webkit-playsinline src="' + videoURL + '" onclick="" controls="controls" preload="metadata"';
+
+    if (posterURL != '') {
```

Comment:
In this call: 
```
mWebView.execJavaScriptFromString("ZSSEditor.insertVideo('" + mediaUrl + "', '" +
    mediaFile.getThumbnailURL() + "', '" + videoPressId +  "');");
```

`mediaFile.getThumbnailURL() ` can be null, so `posterURL != null` is true, and probably showing a broken thumbnail  instead of a placeholder.

BTW, should we add a default placeholder if `posterURL == '' || posterURL == null)`?